vue-frontend/src/components/contact/ThankyouLandingSection.vue

```diff
@@ -74,7 +75,10 @@ export default {
     },
   },
   mounted() {
-    this.name = JSON.parse(localStorage.getItem("name"));
+    if (!this.applicantname && localStorage.getItem("key")) {
+      this.name = JSON.parse(localStorage.getItem("client-name"));
```

Comment:
Use `Dear Applicant` if **applicant-name** is not present in the localstore

Reply:
We shouldn't use `client-name `in place of `applicant-name`. Both are totally different.

Simplify this if-else loop as below: 

- If applicant-name is not present in localstorage, **this.name** will be `Dear applicant`